app/models/container_project.rb

```diff
@@ -76,15 +75,15 @@ def perf_rollup_parents(interval_name = nil)
   alias all_computer_systems    computer_systems
 
   def aggregate_memory(targets = nil)
```

Comment:
possible to swap with virtual total?
(if not, then that is ok)

Reply:
methinks not